packages/graphene.rb

```diff
@@ -0,0 +1,25 @@
+require 'package'
+
+class Graphene < Package
+  description 'A thin layer of graphic data types'
+  homepage 'https://github.com/ebassi/graphene'
+  version '1.6.0'
+  source_url 'https://github.com/ebassi/graphene/archive/1.6.0.tar.gz'
+  source_sha256 '98970f859e452ce421b72726ca727fdf3ac27cb4804b62bfe520157fa46aa2fd'
+
+  depends_on 'ninja'
+  depends_on 'python3'
+  depends_on 'gobject_introspection'
+
+  def self.build
+    system "pip3 install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR} meson"    # depends_on meson
```

Comment:
I'm thinking meson is a good candidate for a separate package.  I can see this being used as a dependency for other packages as well.  Also, isolating it out as a separate package helps with maintenance.  There is no way to uninstall meson if you install it in this manner.

Reply:
How to crea a separate package for meson? @uberhacker

Reply:
Would this work?
```
require 'package'

class Meson < Package
  description 'The Meson Build System'
  homepage 'http://mesonbuild.com/'
  version '0.44.0'
  source_url 'https://github.com/mesonbuild/meson/releases/download/0.44.0/meson-0.44.0.tar.gz'
  source_sha256 '50f9b12b77272ef6ab064d26b7e06667f07fa9f931e6a20942bba2216ba4281b'

  depends_on 'python3'

  def self.install
    system "pip3 install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR} meson"
  end
end
```
You might also want to take a look at https://github.com/mesonbuild/meson to see if it would be worth compiling from source.  Either way would be fine, I think.

Reply:
It works